src/main/java/com/itimpulse/urlshortener/config/OpenApiConfig.java

```diff
@@ -3,43 +3,41 @@
 import io.swagger.v3.oas.models.OpenAPI;
```

Comment:
remove the OpenAPIConfig.java class, it's not needed by the library use have used